pkgs/development/tools/sca2d/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, python3
+, fetchFromGitLab
+, fetchFromGitHub
+}:
+let
+  lark010 = python3.pkgs.lark.overrideAttrs (old: rec {
```

Comment:
This just be fine via python package overwrites otherwise it is not reliable

Reply:
Sorry I didn't understand your point ? I took gdtoolkit (https://github.com/NixOS/nixpkgs/blob/798fd6a3ffb7fae359f6e02a379bfb188d54fcad/pkgs/development/tools/gdtoolkit/default.nix#L7-L17) as example

Reply:
@SuperSandro2000 are you taking about using [`overridePythonAttrs`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/python/python-packages-base.nix#L16-L34) ?

Reply:
yes, https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#overriding-python-packages-overriding-python-packages

Reply:
@SuperSandro2000 Done!